packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -56,9 +56,10 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
     ...other
   } = props;
   const [tabbable, setTabbable] = React.useState(null);
-  const [focused, setFocused] = React.useState(null);
+  const [focusedNodeId, setFocusedNodeId] = React.useState(null);
```

Comment:
`focused` implies that this component is focused but it's actually for tracking other components. For example `FormControl` also has a `focused` state but that has a different meaning.